.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "test/test_helper/bats-assert"]
+	path = test/test_helper/bats-assert
+	url = https://github.com/ztombol/bats-assert.git
```

Comment:
This isn't the right repo. It should be https://github.com/bats-core/bats-assert .